code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -207,6 +207,15 @@
 	do_sparks(3, 1, src)
 	..()
 
+///Disassembling the bot in a civilized manner with a multitool
+/mob/living/simple_animal/bot/cleanbot/disassemble()
+	on = FALSE
+	var/turf/Tsec = get_turf(src)
```

Comment:
Can you rename this to something like `disassemble_turf`?

Reply:
Done!